handler.rb

```diff
@@ -0,0 +1,111 @@
+class Handler
+  def initialize
+    @books = []
+    @persons = []
+    @rentals = []
+  end
+
+  def list_all_books
+    puts 'Database is empty! Add a book.' if @books.empty?
+    @books.each { |book| puts "[Book] Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def list_all_persons
+    puts 'Database is empty! Add a person.' if @persons.empty?
+    @persons.each do |person|
+      puts "[#{person.class.name}] Name: #{person.name}, Age: #{person.age}, id: #{person.id}"
+    end
+  end
+
+  def create_person
+    print 'press 1 to create a student, press 2 to create a teacher : '
+    option = gets.chomp
+
+    case option
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'Invalid input. Try again'
+    end
+  end
```

Comment:
- [ ] When I'm trying to create a person(student or teacher) getting the below error, Please make sure to fix this issue. The app should work as expected after refactoring the code.

![image](https://user-images.githubusercontent.com/83803414/211379739-10d4abe5-cd53-4d50-9aff-67af023980f0.png)